### PR TITLE
Bump OpenTracing to 0.32.0

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

**Comment by pavolloffay** [comment]
Once merged I will release it as 0.3.0

File: src/main/java/io/opentracing/contrib/concurrent/TracedExecutorService.java [hunk 1]
```diff
@@ -59,101 +60,108 @@ public boolean awaitTermination(long l, TimeUnit timeUnit) throws InterruptedExc
...
   @Override
   public Future<?> submit(Runnable runnable) {
-    Scope scope = createScope("submit");
+    Span span = createSpan("submit");
     try {
-      return delegate.submit(tracer.activeSpan() == null ? runnable :
-          new TracedRunnable(runnable, tracer));
+      Span toActivate = span != null ? span : tracer.activeSpan();
+      return delegate.submit(toActivate == null ? runnable :
+          new TracedRunnable(runnable, tracer), toActivate);
```
Comment (objectiser) [inline src/main/java/io/opentracing/contrib/concurrent/TracedExecutorService.java]: The `toActivate` should be in the `TracedRunnable` parameters I believe?

**Review by objectiser — APPROVED** [review]
LGTM - just the one issue.

File: src/main/java/io/opentracing/contrib/concurrent/TracedExecutorService.java (hunk 1 above)
Comment (pavolloffay) [reply src/main/java/io/opentracing/contrib/concurrent/TracedExecutorService.java]: good catch!

File: src/main/java/io/opentracing/contrib/concurrent/TracedExecutor.java [hunk 2]
```diff
@@ -28,20 +28,21 @@ public TracedExecutor(Executor executor, Tracer tracer, boolean traceWithActiveS
...
-        scope.close();
+      // close the span if created
+      if (span != null) {
+        span.finish();
       }
     }
   }
 
-  Scope createScope(String operationName) {
+  Span createSpan(String operationName) {
```
Comment (sjoerdtalsma) [inline src/main/java/io/opentracing/contrib/concurrent/TracedExecutor.java:43]: Nit: The method name does not indicate that there may _not_ be a span created (which in cases with active spans would actually be the norm).

The name should make it feel natural that `null` gets returned and in which situations. Something like `createNewSpanWhenNoActiveSpanExists()` may be a bit too verbose, but I hope you get my point.
This rename may be a good time to think of a better name.

**Review by sjoerdtalsma — APPROVED** [review]
Aside from the naming comment, functionally this looks good!

File: src/main/java/io/opentracing/contrib/concurrent/TracedExecutor.java (hunk 2 above)
Comment (pavolloffay) [reply src/main/java/io/opentracing/contrib/concurrent/TracedExecutor.java:43]: thanks @sjoerdtalsma I will leave it as it was. The method also takes into account an internal parameter whether to create a span or not.

**Pull request merged**